xla/service/gpu/gpu_compiler.cc

```diff
@@ -1412,8 +1412,9 @@ absl::Status GpuCompiler::OptimizeHloPostLayoutAssignment(
     // ReductionDimensionGrouper, as that makes matching the softmax pattern
     // harder.
     if (debug_options.xla_gpu_enable_triton_softmax_fusion() &&
-        cuda_cc != nullptr &&
-        cuda_cc->IsAtLeast(se::CudaComputeCapability::AMPERE)) {
+        ((cuda_cc != nullptr &&
+        cuda_cc->IsAtLeast(se::CudaComputeCapability::AMPERE)) ||
+        rocm_cc != nullptr)) {
```

Comment:
This change enable Triton-based codegen for SoftMax (and other normalization layers) for ROCm. Is that the intention?
Please note that we will `xla_gpu_enable_triton_softmax_fusion` to default to `true` at some point. Will this work for all AMDGPU architectures?

Reply:
Yes, it was intention to enable Triton-based codegen for SoftMax (for all AMDGPU architectures).